cyberark/summon
# fix: replace conjurinc with cyberark + move github pages from branch to /docs

Addresses https://github.com/cyberark/summon/issues/46 https://github.com/cyberark/summon/issues/47

#### What does this pull request do?
+ Updates relevant references to the `conjurinc` github organisation to `cyberark`
+ Migrates `githubpages` branch to `/docs` folder in root of repository
  + Once merged an admin needs to modify the settings such that the GitHub Pages publishing source is the master branch /docs folder as opposed to the gh-pages branch.

#### How to I test this PR ?

+ I forked this repository and merged the PR into the fork. Then I modified the settings such that the GitHub Pages publishing source is the master branch /docs folder as opposed to the gh-pages branch. The result is https://doodlesbykumbi.github.io/summon/
+ Use http://www.deadlinkchecker.com/website-dead-link-checker.asp or any other dead link checker to ensure there are no broken links on the result above
+ Jenkins build should be green
